docs/specs/om/open_metrics_spec_2_0.md

```diff
@@ -809,15 +936,78 @@ foo_count 17 ct@1520430000.123
 foo_sum 324789.3 ct@1520430000.123
 ```
 
+##### Histogram with native buckets
+
+The MetricPoint's value MUST be a complex data type.
+
+Histograms with Native Buckets MUST use the integer native histogram data type.
+
+The integer native histogram data type is represented as structured data with fields. There MUST NOT be any whitespace around fields.
+The integer native histogram data type MUST include the Count, Sum, Schema, Zero Threshold, Zero Native Bucket value as the fields `count`, `sum`, `schema`, `zero_threshold`, `zero_count`, in this order.
+
+If there are no negative Native Buckets, then the fields `negative_spans` and `negative_buckets` SHOULD be omitted.
+If there are no positive Native Buckets, then the fields `positive_spans` and `positive_buckets` SHOULD be omitted.
+
+If there are negative (and/or positive) Native Buckets, then the fields `negative_spans`, `negative_buckets` (and/or `positive_spans`, `positive_buckets`) MUST be present in this order after the `zero_count` field.
+
+With the exception of the `sum` and `zero_threshold` field, all numbers MUST be integers and MUST NOT include dot '.' or exponent 'e'.
+
+Native Bucket values MUST be ordered by their index, and their values MUST be placed in the `negative_buckets` (and/or `positive_buckets`) fields.
```

Comment:
note to reviewers: we don't use delta encoding here, `negative_buckets` and `positive_buckets` hold the actual absolute non negative integer value.